Features/CopyMoveDelete/CopyContent.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Copy content using te role of Editor
+    In order to validate the copy action
+    As an Editor user
+    I need to be able to copy an object that I am viewing
+
+    Background:
+        Given I am logged in as an Editor in PlatformUI
+
+    ##
+    #Copy objects to valid locations
+    ##
+    @javascript
+    Scenario: Copy one object without children objects
+        Given a "Destiny" folder exists
+        And a "News Flash" article exists
+        When I copy the "News Flash" into "Destiny" folder
```

Comment:
I might be off-topic, but I would have expected more details on the action of copying, from a user's perspective. The scenario does not say which page/view I'm standing on, and what I do to copy.